doc/source/examples/incompressible-flow/2d-naca0012-low-reynolds/2d-naca0012-low-reynolds.rst

```diff
@@ -0,0 +1,301 @@
+        
+.. image:: image/ClCdResultsPlot.png
+
+There is a slight variation of the results obtained compared to the litterature results. Indeed, the simulation is purely two-dimensionnal while Kouser et al. take a 2D slice of a 3D simulation. Still, the evolution observed is relevant.
+
+One can also see the low velocity zones on the upper part of the airfoil which corresponds to the boundary layer : the ``noslip`` condition on the NACA imposes a zero velocity condition on the fluid. This create a zone where the advective and diffusive forces are in competition. The following streamline representation helps to see the movements of the fluid inside the boundary layer : 
+
+.. image:: image/nacaStreamlineAngles.png
+
+It can be observed that zones of recirculation form on the airfoil. This is due to two phenomenas : first the flow outside of the boundary layer tends to "pull" it in its direction and the ``noslip`` boundary condition slows the fluid, then a positive pressure gradient -commonly refered to as adverse pressure gradient- on the airfoil surface pushes the fluid backwards. Following this a vertex forms. Below is represented the mean pressure coefficient :math:`C_p` on the airfoil with comparison to the litterature right below. It is computed using the following formula : 
+
+.. math::
+        C_p = \frac{p-p_{\infty}}{0.5\rho_{\infty}u_{\infty}^2}
+        
+with :math:`p_{\infty}` the static pressure in the freestream (equal to 0 in this case), :math:`\rho_{\infty}` the freestream fluid density, equal to the fluid density since we are solving an incompressible flow and :math:`u_{\infty}` the freestream velocity of the fluid, equal to ``1.0`` in this case.
+
+.. image:: image/CpPlot.png
+
+.. image:: image/cpPlotRef.png
+
+The important pressure at the attacking edge of the airfoil is what allows the incoming flow to be deflected to the upper and lower part. Then, if we look at the upper part (be careful about the reversed y-axis since :math:`-C_p` is plotted) the adverse pressure gradient is visible. Then at the trailing edge, the mesh is ot precise enough. A slight discontuinity ensues which, though not physically accurate, do not invalidate the whole result.
+
+
+For angles of attack :math:`\alpha\geq 9°`, the vortex start to detach from the airfoil, this can be seen using the instantaneous velocity fields. For each angle of attack, the last time iteration was taken :  
+
+.. image:: image/vI.png
+
+In order to retrieve the frequency of the vortex shedding, one can look at the variations of :math:`C_L`. Below are the variations of the lift coefficient with time, only the case where :math:`\alpha=15°` was considered for this analysis : 
```

Comment:
```suggestion
In order to retrieve the frequency of the vortex shedding, one can look at the variations of :math:`C_L`. Below are the variations of the lift coefficient with time for the case where :math:`\alpha=15°` was considered for this analysis : 
```

Reply:
variations could also be replaced by fluctuations